CTP/shell/src/com/navercorp/cubridqa/shell/common/CommonUtils.java

```diff
@@ -280,6 +283,25 @@ public static String toFixedLength(String str, int len, char fillChar) {
 		return result.substring(result.length() - len);
 	}
 	
+	public static void generateFailBackupPackage(Context context){
+		if(context.isWindows()) return;
+		String backupFileName = "";
+		Calendar cal = Calendar.getInstance();
+		int year = cal.get(Calendar.YEAR);  
+        int month = cal.get(Calendar.MONTH) + 1;
+        int day = cal.get(Calendar.DAY_OF_MONTH);  
+        int hour = cal.get(Calendar.HOUR);  
+        int minute = cal.get(Calendar.MINUTE);  
+        int second = cal.get(Calendar.SECOND);
+        String curTimestamp = year + "." + month + "." + day + "_" + hour + "." + minute + "." + second;
+        String build = context.getTestBuild();
+        String bit = context.getVersion();
+        Integer tastId = context.getTaskId();
```

Comment:
as is:
 Integer tastId = context.getTaskId();

There is typo in 'tastId'.

And it's better to apply the code format.